packages/ember-environment/lib/global.js

```diff
@@ -14,5 +14,4 @@ function checkElementIdShadowing(value) {
 export default checkGlobal(checkElementIdShadowing(typeof global === 'object' && global)) ||
                checkGlobal(typeof self === 'object' && self) ||
                checkGlobal(typeof window === 'object' && window) ||
-               mainContext || // set before strict mode in Ember loader/wrapper
```

Comment:
Sorry about that. Can we add a comment to it this time describing what the unused variable does?